mlflow/pipelines/steps/train.py

```diff
@@ -586,6 +653,17 @@ def objective(hyperparameter_args):
                         "log_model_explainability": False,
                     },
                 )
+                autologged_params = mlflow.get_run(run_id=tuning_run.info.run_id).data.params
+                for param_name, param_value in estimator_args.items():
+                    if param_name in autologged_params:
+                        if not self._is_equal(param_value, autologged_params[param_name]):
+                            _logger.warning(
+                                f"Failed to log parameter {param_name} due to "
+                                f"conflict. old_value: {autologged_params[param_name]} "
+                                f"new_value: {param_value} type: {type(param_value)}"
+                            )
+                    else:
+                        mlflow.log_param(param_name, param_value)
```

Comment:
Can we batch up the parameters that we want to log (the ones without conflicts) and make a single `mlflow.log_params()` call? This is more efficient, especially with API rate limits enforced on Databricks